getsolus/packages
# Split pygobject

**Summary**

The upcoming python-gobject 3.48.1 is now built with meson and it's pkgconfig file is no longer compatible. Let's split ahead of that release.

**Test Plan**

- Add as local repo, upgrade, verify solus-sc and doflicky were still working as expected.
- Verify conflicts on -devel subpackage work

**Checklist**
